pom.xml

```diff
@@ -44,7 +44,7 @@
     <dependency>
       <groupId>org.mockito</groupId>
       <artifactId>mockito-core</artifactId>
-      <version>4.11.0</version>
+      <version>5.11.0</version>
```

Comment:
I am not sure why you are upgrading this dependency; the pr has no use of this plugin.

Reply:
Worse, it's not compatible with Java 8 🙁